install.sh

```diff
@@ -35,16 +42,90 @@ fi
 sudo mkdir -p "$INTERNAL_DIR"
 sudo mkdir -p "/tmp/roller_tmp"
 
+# Function to display Go installation instructions
+GO_REQUIRED_VERSION="1.19"
+display_go_installation_instructions() {
+    echo "$EMOJI To install Go $GO_REQUIRED_VERSION, you can run the following commands:"
+    echo "  wget https://go.dev/dl/go1.19.10.${OS}-${ARCH}.tar.gz"
+    echo "  sudo rm -rf /usr/local/go && sudo tar -C /usr/local -xzf go1.19.10.${OS}-${ARCH}.tar.gz"
+}
+
 # Download and extract the tar file to a temporary directory
 echo "$EMOJI Downloading roller..."
-sudo curl -L "$TGZ_URL" --progress-bar | sudo tar -xz -C "/tmp/roller_tmp"
-
-# Assuming that the tar file contains the lib folder and the roller and rollapp_evm binaries inside the roller_bins directory.
-# Move binaries to their correct locations
-echo "$EMOJI Installing roller..."
-sudo mv "/tmp/roller_tmp/roller_bins/lib"/* "$INTERNAL_DIR"
-sudo mv "/tmp/roller_tmp/roller_bins/roller" "$ROLLER_BIN_PATH"
-sudo mv "/tmp/roller_tmp/roller_bins/rollapp_evm" "$ROLLAPP_EVM_PATH"  # move the rollapp_evm binary
+if ! sudo curl -L "$TGZ_URL" --progress-bar | sudo tar -xz -C "/tmp/roller_tmp"; then
```

Comment:
Consider removing the negation operator (!) from the condition check and flipping the logic inside the if block.